openassessment/staffgrader/staff_grader_mixin.py

```diff
@@ -194,6 +199,39 @@ def list_staff_workflows(self, data, suffix=''):  # pylint: disable=unused-argum
                 log.exception("Failed to serialize workflow %d: %s", staff_workflow.id, str(e), exc_info=True)
         return result
 
+
+
+    @XBlock.json_handler
+    @require_course_staff("STUDENT_GRADE")
+    def list_assessments(self, data, suffix=''):  # pylint: disable=unused-argument
+        """
+        List the assessments' grades based on the type (received or given) for a specific submission.
+
+        Args:
+            data (dict): Contains the necessary information to fetch the assessments.
+                        - 'item_id': The ID of the xblock/item.
+                        - 'submission_uuid': The UUID of the submission.
+                        - 'assessment_type': A string, either "received" or any other value 
+                                            to determine the type of assessments to retrieve.
+
+        Returns:
+            list[dict]: A list of dictionaries, each representing an assessment's data.
+
+        Note:
+            - If 'assessment_type' is "received", the function fetches assessments received 
```

Comment:
In the ora context assessment_type means other things: https://github.com/openedx/edx-ora2/blob/77c2dbe104b032f716a1dd5fb21ae60761a859f9/test/acceptance/pages.py#L422

Reply:
changed to `assessment_filter`